modules/installation-user-infra-machines-advanced.adoc

```diff
@@ -83,11 +83,11 @@ For example:
 
 * `/var/lib/containers`: Holds container-related content that can grow
 as more images and containers are added to a system.
-* `/var/log`: Holds logging data that you might want to keep separate for
-auditing purposes later.
+* `/var`: Holds data that you might want to keep separate for purposes such as
```

Comment:
I would re-write the end of this sentence, the word "later" at the end feels out of place now.

"Holds data that you might want to keep separate for future purposes, such as auditing."